src/helpers/exapleErrors.ts

```diff
@@ -0,0 +1,54 @@
+import { ErrorList } from '../types/ErrorList'
+
+export const exampleErrors = {
```

Comment:
Maybe we should move it somewhere to the test deir? It's strange that error examples in helpers

Reply:
The jest will try to test the file than